paz/backend/boxes.py

```diff
@@ -5,60 +5,47 @@
 
 
 def split(boxes):
+    """Split boxes into x_min, y_min, x_max, y_max components."""
     return jp.split(boxes, 4, axis=1)
 
 
-def join(coordinate_0, coordinate_1, coordinate_2, coordinate_3):
-    return jp.concatenate(
-        [coordinate_0, coordinate_1, coordinate_2, coordinate_3], axis=1
-    )
-
-
-def to_center_form(boxes):
-    """Transform from corner coordinates to center coordinates.
-
-    # Arguments
-        boxes: Numpy array with shape `(num_boxes, 4)`.
-
-    # Returns
-        Numpy array with shape `(num_boxes, 4)`.
-    """
-    x_min, y_min, x_max, y_max = split(boxes)
+def compute_centers(x_min, x_max, y_min, y_max):
+    """Compute center coordinates of boxes."""
     center_x = (x_max + x_min) / 2.0
     center_y = (y_max + y_min) / 2.0
-    W = x_max - x_min
-    H = y_max - y_min
-    return jp.concatenate([center_x, center_y, W, H], axis=1)
-
+    return center_x, center_y
 
-def to_xywh(boxes):
-    return xyxy_to_xywh(boxes)
 
-
-def xyxy_to_xywh(boxes):
-    x_min, y_min = boxes[:, 0:1], boxes[:, 1:2]
-    x_max, y_max = boxes[:, 2:3], boxes[:, 3:4]
+def compute_size(x_min, x_max, y_min, y_max):
```

Comment:
This function should take as input boxes not the coordinates. If you are using this function as coordinates, consider splitting the boxes first the applying this function. If you are using this function and the coordinates are scalar values. Then  , let's define the function inside the function that is using it.